tests/fixtures/MailsMockTraitTest/test_mail_with_global_export.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
you using this fixture to check that globa export mode works fine and create file if it not exists. If you'll commit this file to git - test will always be successful completed even if global export mode will be broken